docs/UserGuide.md

```diff
@@ -9,331 +9,424 @@ Interface** (CLI) while still providing an easy way to visualize all events thro
 
-[[Back to top](#index)]
+<div markdown="span" class="alert alert-warning">
+:warning: **Warning:** Index must be a positive integer and exist in the scheduler.
+</div>
 
+[[Back to top](#table-of-contents)]
 
-<h3 id="list">Listing All Events</h3>
+<h3 id="find">Locate events `find`</h3>
 
-Displays all events currently saved in the Scheduler.
+Search the scheduler and returns all events that matches the name, date or a combination of both.
 
-Format: `list`
+Format:`find n/<name> d/<date>`  
+Example: `find n/tennis` or `find d/2023-03-01` or `find n/tennis d/2023-03-01`
 
-Example: `list`
+Parameters:  
+`n/`: Name of the event  
+`d/`: Date of the event
 
-[[Back to top](#index)]
+<div markdown="span" class="alert alert-info">
+:information_source: **Note:**  
+Name is alphanumeric only.
+Partially matching names will also be considered as a match.  
+Date should follow a `YYYY-MM-DD` format.
+</div>
+
+<div markdown="span" class="alert alert-warning">
+:warning: **Warning:** At least one field must be provided.
+</div>
 
+[[Back to top](#table-of-contents)]
 
-<h3 id="next">Retrieve next event</h3>
+<h3 id="next">Retrieve next events `next`</h3>
 
 Shows the next upcoming event(s).
 
-Format: `next <count>`
+Format:`next <count>`  
+Example: `next` or `next 2`
 
 Parameter:  
 `<count>`: Number of upcoming events to show.
-           Optional - if not indicated, only the next one event is shown.
+**Optional** - if not indicated, only the next one event is shown.
 
 <div markdown="span" class="alert alert-info">
 :information_source: **Note:** If count is larger than the number of existing events, then all events are shown.
 </div>
 
-Example: `next` or `next 2`
-
-[[Back to top](#index)]
+[[Back to top](#table-of-contents)]
 
+<h3 id="undo">Undo previous command</h3>
 
-<h3 id="find">Locate events by name and/or date</h3>
+Undo the previously executed valid command.
 
-Finds the Scheduler for the task, returns all events containing the relevant keyword.
-
-Format:`find n/<name> d/<date>`
-
-Parameters:  
-`n/`: Name or description of event  
-`d/`: Date of the event
+Format: `undo`  
+Example: `undo`
 
 <div markdown="span" class="alert alert-warning">
-:warning: **Warning:** At least one field must be provided.
+:warning: **Warning:** `undo` is only applicable for `add`, `delete`, `edit` and `recur` commands.
 </div>
 
-Example: `find n/tennis` or `find d/2023-03-01` or `find n/tennis d/2023-03-01`
-
-[[Back to top](#index)]
-
-
-<h3 id="undo">Undo Previous Command</h3>
+[[Back to top](#table-of-contents)]
 
-Undo previous valid command.
-(Only applicable for `add`, `delete`, `edit` and `recur` commands).
+<h3 id="list">Listing all events</h3>
 
-Format: `undo`
+Displays all events currently saved in the scheduler.
 
-Example: `undo`<br><br>
-
-[[Back to top](#index)]
+Format: `list`  
+Example: `list`
 
+[[Back to top](#table-of-contents)]
 
-<h3 id="help">View Help</h3>
+<h3 id="help">View help</h3>
```

Comment:
Since this is a header, I feel that using [Title Case](https://en.wikipedia.org/wiki/Title_case) would be appropriate.
> When using title case, all words are capitalized, except for minor words (typically articles, short prepositions, and some conjunctions) that are not the first or last word of the title.